apps/src/p5lab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -146,19 +146,39 @@ export default class AnimationPickerBody extends React.Component {
     if (this.props.hideBackgrounds) {
       categories = categories.filter(category => category !== 'backgrounds');
     }
+    // Level-specific animations currently have the following categories:
+    // "level_animations", "backgrounds", "animals".
+    // We want to hide the "animals" category which has only one animation in it.
+    // It will be displayed in the "all" category added below.
+    // There is an 'animals' category because a level-specific animation was uploaded with the
+    // 'animals' category BEFORE the sprite upload feature dictated that level-specific animations
+    // can only be categorized as "costume" or "background".
+    if (categories.includes('level_animations')) {
+      categories = categories.filter(
+        category =>
+          category === 'level_animations' || category === 'backgrounds'
+      );
+    }
     categories.push('all');
-    return categories.map(category => (
-      <AnimationPickerListItem
-        key={category}
-        label={
-          msg[`animationCategory_${category}`]
-            ? msg[`animationCategory_${category}`]()
-            : category
-        }
-        category={category}
-        onClick={this.onCategoryChange}
-      />
-    ));
+    return categories.map(category => {
+      let label = msg[`animationCategory_${category}`]
+        ? msg[`animationCategory_${category}`]()
+        : category;
+      if (label === 'Level Animations') {
+        label = 'Costumes'; // This is a more accurate label for level-specific animations that are not backgrounds.
+      }
```

Comment:
>Instead of doing this, would it be better to do an earlier check on category to map level_animations -> costumes? I'm not sure if that's a valid key. This way is just changing the label but keeping the key as level_animations. Can we add a costumes key?

I hesitated doing this because the costume animations are saved with the 'level_animations' category. But I found where we use the `categoryQuery` [here](https://github.com/code-dot-org/code-dot-org/blob/9518a6194c719001120f79e39129b5334add695e/apps/src/p5lab/AnimationPicker/AnimationPickerBody.jsx#L77) so just added a condition.
Lmk what you think - thanks!

Reply:
>Also, both this label and the one down below on 275 are duplicated strings, and are they getting localized properly?

I think this [levelbuilder page](https://levelbuilder-studio.code.org/sprites/select_start_animations?library=all) is not localized since it is an internal tool for content-editors. 'Level-specific' animations are effectively hidden from users.